cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -1608,4 +1666,75 @@ private Map<Long, Long> getOperationsCountMapBySequenceNumber(List<DurableLogIns
                 originalOperations.stream().collect(Collectors.groupingBy(op -> op.getSequenceNumber(), Collectors.counting()));
         return resultMap;
     }
+
+    /*
+    * EntrySerializer's serializeUpdateWithExplicitVersion has package access and changing the scope for test
```

Comment:
Please, rephrase the comment to something like: "Create a test version of EntrySerializer to serialize Table Segment entry for recovery purposes without changing visibility of of methods from original class.". I don't think it is necessary to explain the coverage issues here, as they are too specific of the build configuration we have.

Reply:
Updated.